### ISSUE TITLE
Webstorm - Typescript error on the able pipe

### ISSUE THREAD

**Issue by twmobius** [stalniy/casl-examples#14]
Hi there,

first off, thanks for casl, it's a great library. I've noticed that WebStorm 2020.2 complains on the able pipe even on the angular-todo example app.

![Screenshot 2021-05-13 at 12 55 02 PM](https://user-images.githubusercontent.com/854155/118109939-787b5580-b3ea-11eb-881e-121b2aa2ffb0.png)

The application compiles properly, but there seems to be an issue with the typescript checking on the IDE. I would be very much obliged if you could look into it

**Comment by stalniy** [comment]
The issue is that I do t use Webstorm) very likely it’s somehow related to language service. Do you have some plugin for Angular?
But I can check in vscode though

**Comment by twmobius** [comment]
WebStorm comes with Angular plugin bundled. I haven't installed anything that could interfere with typescript and or angular definitions. 

I will check it with the latest WebStorm version to see if the problem still exists and get back to you

**Comment by twmobius** [comment]
Yes I can confirm that the same error occurs on WebStorm 2021.1.1 (latest release). I am pretty sure this has something to do with the way the typescript definition for the pipe is being generated.

**Comment by fredjammes** [comment]
Same here with WebStorm 2021.2.2.
I tried to understand the type of the pipe transform args but I never saw this syntax :
    transform(...args: Parameters<T['can']>): boolean;
Does it means it is parameters of the generic type T which contains a property called can ?
